.lintstagedrc

```diff
@@ -1,6 +1,6 @@
 {
   "*.{js,ts}": [
-    "eslint --fix",
+    "eslint --max-warnings 0 --fix",
```

Comment:
We are setting max-warnings to 0 in the CI so I think it's better to have same conditions in precommit hook.